package.json

```diff
@@ -58,7 +60,7 @@
     "babel-preset-react": "^6.24.1",
     "commander": "^2.15.1",
     "glob": "^7.1.2",
-    "jsdom": "^12.0.0",
```

Comment:
I think this might be challenging for us to land at Airbnb, so it might be nice to allow v12 though v16 for a while to make it easier to get here.

Reply:
I'm actually not sure how to do that without making jsdom a peer dependency (which will lead to other issues).

Reply:
I just tried the release candidate and the update worked smoothly!